File: eunbi/week4/숫자문자열과영단어.java
package algorithmStudy;

import java.util.*;

class Solution {
    public int solution(String s) {
        String[] number = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
        
        for(int i=0; i<10; i++)
            s = s.replaceAll(number[i], Integer.toString(i));

        return Integer.parseInt(s);
    }
}

File: seoyeon/week4/숫자 문자열과 영단어.py
def solution(s):
    number = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
    for i in range(len(number)) :
        s = s.replace(number[i], str(i))
    return int(s)